tests/test_redact.py

```diff
@@ -22,12 +23,18 @@ def svs_input_path(data_dir, request) -> Path:
     return data_dir / "input" / request.param
 
 
+@freeze_time("2023-05-12 12:12:53")
+def test_create_redact_dir(tmp_path):
```

Comment:
I don't necessarily think it's required now, but this unit test should hopefully make it easy to test that permission errors are handled (and knowing exactly what sort of filesystem states actually cause permission errors).

Reply:
use this function to address #58 (in future PR)